packages/asc-ui/src/components/LinkList/LinkListItemStyle.ts

```diff
@@ -4,7 +4,11 @@ import { color, svgFill } from '../../utils'
 import IconStyle from '../Icon/IconStyle'
 import TypographyStyle from '../Typography/TypographyStyle'
 
-export default styled.li`
+export type Props = {
+  className?: string
+}
+
+export default styled.li<Props>`
```

Comment:
Use `React.HTMLAttributes<HTMLListElement>` here please instead of `className?: string`

Reply:
done